components/RegistrationForm/RegistrationForm.js

```diff
@@ -28,8 +28,12 @@ const registrationSchema = Yup.object().shape({
   'confirm-password': Yup.string()
     .required(validationErrorMessages.required)
     .oneOf([Yup.ref('password')], validationErrorMessages.passwordMatch),
-  firstName: Yup.string().required(validationErrorMessages.required),
-  lastName: Yup.string().required(validationErrorMessages.required),
+  firstName: Yup.string()
+    .required(validationErrorMessages.required)
+    .matches(/^[a-zA-Z]+$/, validationErrorMessages.name),
```

Comment:
So names _can_ have symbols.

Check this out https://www.regextester.com/93648

Reply:
I was trying to figure out a good way to do this. I'm not too well versed in regex. Can I just do: exclude numbers and exclude special characters but allow some special character group? I'm assuming that's what the linked example does.

I'm thinking something like this maybe   `^[a-zA-Zà-žÀ-Ž \-]+$`

Reply:
Still missing apostrophes and commas. Consider copy-pasting the regex! That’s basically what I do at work 😂

Reply:
```/^[a-z ,.'-]+$/i```

Reply:
``` /^[a-zà-ž ,'`-]+$/i ```  I had settled on this earlier as it allows northern wandering types or just names with accents and the like. But I can use your version as well.  


<img width="626" alt="Screen Shot 2019-06-16 at 9 17 26 PM" src="https://user-images.githubusercontent.com/10247647/59578280-581ab300-907c-11e9-96a5-18597e7f09f2.png">